bundles/org.openhab.automation.jsscripting/README.md

```diff
@@ -21,23 +36,215 @@ log:set DEBUG org.openhab.core.automation
+
+// Get response from command line.
+let response = openhab.Exec.executeCommandLine('echo', 'Hello Wordl!');
+
+// Get response from command line with timeout.
+response = openhab.Exec.executeCommandLine(Duration.ofSeconds(20), 'echo', 'Hello Wordl!');
+```
+
+### HTTP Actions
+
+For available actions have a look at the [HTTP Actions Docs](https://www.openhab.org/docs/configuration/actions.html#http-actions).
+
+```javascript
+openhab.HTTP = Java.type('org.openhab.core.model.script.actions.HTTP');
+
+// Example GET Request
+var response = openhab.HTTP.sendHttpGetRequest('<url>');
+```
+
+Replace `<url>` with the request url.
+
+### Timers
+
+```javascript
+let ZonedDateTime = Java.type('java.time.ZonedDateTime');
+let now = ZonedDateTime.now();
+openhab.ScriptExecution = Java.type('org.openhab.core.model.script.actions.ScriptExecution');
+
+// Function to run when the timer goes off.
+function timerOver () {
+  logger.info('The timer is over.');
+}
+
+// Create the Timer.
+this.myTimer = openhab.ScriptExecution.createTimer(now.plusSeconds(10), timerOver);
+
+// Cancel the timer.
+this.myTimer.cancel();
+
+// Check whether the timer is active. Returns true if the timer is active and will be executed as scheduled.
+let active = this.myTimer.isActive();
+
+// Reschedule the timer.
+this.myTimer.reschedule(now.plusSeconds(5));
+```
+
+### Scripts Actions
+
+Call scripts created in the UI (Settings -> Scripts) with or without parameters.
+
+```javascript
+openhab.scriptExtension = Java.type('org.openhab.core.automation.module.script.ScriptExtensionProvider');
+let bundleContext = Java.type('org.osgi.framework.FrameworkUtil').getBundle(openhab.scriptExtension.class).getBundleContext();
+openhab.RuleManager = bundleContext.getService(bundleContext.getServiceReference('org.openhab.core.automation.RuleManager'));
+
+// Simple call.
+openhab.RuleManager.runNow('<scriptToRun>');
+
+// Advanced call with arguments.
+let map = new java.util.HashMap();
+map.put('identifier1', 'value1');
+map.put('identifier2', 'value2');
+// Second argument is whether to consider the conditions, third is a Map<String, Object> (a way to pass data).
+openhab.RuleManager.runNow('<scriptToRun>', true, map);
+```
+
+Replace `<scriptToRun>` with your script's (unique-)id.
+
+## Cloud Notification Actions
+
+Notification actions may be placed in Rules to send alerts to mobile devices registered with an [openHAB Cloud instance](https://github.com/openhab/openhab-cloud) such as [myopenHAB.org](https://myopenhab.org/).
+
+For available actions have a look at the [Cloud Notification Actions Docs](https://www.openhab.org/docs/configuration/actions.html#cloud-notification-actions).
+
+```javascript
+openhab.NotificationAction = Java.type('org.openhab.io.openhabcloud.NotificationAction')
+
+// Example
+openhab.NotificationAction.sendNotification('<email>', '<message>'); // to a single myopenHAB user identified by e-mail
```

Comment:
FYI the helper library will lookup all actions a user may have installed and make those available on the `actions` namespace,  so in this example if the user has the cloud binding installed, it would automatically be available via `actions.NotificationAction.sendNotification`